pkg/test/deployment/util.go

```diff
@@ -54,45 +52,3 @@ func DumpPodState(workDir string, namespace string, accessor *kube.Accessor) {
 		}
 	}
 }
-
-// DumpPodData copies pod logs from Kubernetes to the specified workDir.
-func DumpPodData(workDir, namespace string, accessor *kube.Accessor) {
```

Comment:
Any chance you can keep this around? (Even commented out).

I'll rationalize these methods together soon. There are multiple versions of it.